documentation/meta/traffic/runbooks/identifying-and-blocking-traffic-anomalies.md

```diff
@@ -0,0 +1,161 @@
+# Identifying and Blocking Traffic Anomalies in Cloudflare
+
+## Objective
+
+Enable consistent and prompt responses to traffic anomalies by implementing
+procedures for identifying and blocking suspicious traffic—high request rates,
+high error volume, or scraping patterns which disrupt service—in Openverse.
+
+## When to use this runbook
+
+Use this runbook when either of these are true:
+
+1. There is a service disruption to the Openverse API or frontend.
+2. There is a marked increase in resource usage on the API or frontend ECS
+   services that continues to trend to unsustainable levels, which _could lead_
+   to a service disruption.
+
+And:
+
+1. There is a marked increase in traffic visible in our AWS dashboards or
+   Cloudflare analytics.
```

Comment:
Would it be okay to put the necessary condition first? It's confusing to read in this order:

```suggestion
Use this runbook when there is a marked increase in traffic visible in our AWS dashboards or
   Cloudflare analytics and either of these are true:

1. There is a service disruption to the Openverse API or frontend.
2. There is a marked increase in resource usage on the API or frontend ECS
   services that continues to trend to unsustainable levels, which _could lead_
   to a service disruption.
```